qtpy/QtCore.py

```diff
@@ -24,6 +24,15 @@
     from PyQt6.QtCore import QDateTime
     QDateTime.toPython = QDateTime.toPyDateTime
 
+    # For issue #311
+    # Seems like there is an error with sip. Without first
+    # trying to import `PyQt6.QtGui.Qt`
+    # Some functions like `PyQt6.QtCore.Qt.mightBeRichText` are missing
```

Comment:
```suggestion
    # Seems like there is an error with sip. Without first
    # trying to import `PyQt6.QtGui.Qt`, some functions like
    # `PyQt6.QtCore.Qt.mightBeRichText` are missing.
```

Fix punctuation/caps